allennlp/models/bert_for_classification.py

```diff
@@ -0,0 +1,141 @@
+from typing import Dict, Union
+
+from overrides import overrides
+import torch
+from pytorch_pretrained_bert.modeling import BertModel
+
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.model import Model
+from allennlp.nn.initializers import InitializerApplicator
+from allennlp.training.metrics import CategoricalAccuracy
+
+
+@Model.register("bert_for_classification")
+class BertForClassification(Model):
```

Comment:
So, when we talked about this earlier, I was imagining a `Module`, not a `Model`.  But, this is probably better, because it's simpler and still gives someone an easy way to do this with allennlp if they want to.  It's just an example model that does classification with BERT, in a BERT-specific way.